src/main/java/qna/domain/Answer.java

```diff
@@ -24,11 +30,13 @@
     @Column(nullable = false)
     private boolean deleted = false;
 
-    @Column
-    private Long questionId;
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @JoinColumn(foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```

Comment:
기본 값이 있지만 외래키 이름에 대한 부분을 명확히 작성해주시는 것도 좋을 것 같아요
```suggestion
    @JoinColumn(name="question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
```